### ISSUE TITLE
chrome://accessibility Not working in Electron 5+

### ISSUE THREAD

**Issue by cifratila** [electron/electron#24403]
### Preflight Checklist

* [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [X] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 5, 6, 7, 8, 9
* **Operating System:**
  * All platforms
* **Last Known Working Electron version:**
  * 4.2.12

### Expected Behavior
The chrome://accessibility page loads correctly in a BrowserWindow
<img width="797" alt="Screen Shot 2020-07-03 at 7 55 23 AM" src="https://user-images.githubusercontent.com/282777/86436480-a12ebb00-bd02-11ea-9106-b34ee94a98fe.png">

### Actual Behavior
The chrome://accessibility page is blank
<img width="799" alt="Screen Shot 2020-07-03 at 7 54 54 AM" src="https://user-images.githubusercontent.com/282777/86436503-adb31380-bd02-11ea-909e-af3e9a4f2936.png">

### To Reproduce
https://gist.github.com/cifratila/a2621c409a05ed034b95662566471675
You can run this in electron-fiddle

**Comment by cifratila** [comment]
As discussed with @deepak1556 
quick answer the relevant webui page resources have been moved out of //content to //chrome https://chromium-review.googlesource.com/c/chromium/src/+/1144187/ , fix should be fairly simple by including those sources again from //chrome
